salt/modules/pkgin.py

```diff
@@ -98,6 +105,7 @@ def __virtual__():
 
 
 def _splitpkg(name):
+    '''Split package name from versioned string'''
```

Comment:
Can you split this docstring into 3 separate lines?

Reply:
Done, I would have expected lint to catch these as wel though?